src/test/java/pageFactory/baseEntities/BasePages.java

```diff
@@ -43,5 +51,16 @@ protected void waitForOpen() {
             throw new RuntimeException("Page was not opened");
         }
     }
+    public void screenshot(){
+        File screenshot = ((TakesScreenshot)driver).getScreenshotAs(OutputType.FILE);
+        Date dateNow = new Date();
+        SimpleDateFormat format = new SimpleDateFormat("d MMM yyyy hh-mm-ss");
+        String fileName = format.format(dateNow) + ".png";
 
+        try {
+            FileUtils.copyFile(screenshot, new File("D://Screenshots//" + fileName));
```

Comment:
you can get screenshot as bytes or smthing similar, and not create 2 files for that